tests/BenchmarkDotNet.IntegrationTests.ManualRunning.MultipleFrameworks/BenchmarkDotNet.IntegrationTests.ManualRunning.MultipleFrameworks.csproj

```diff
@@ -17,6 +17,8 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugSymbols>true</DebugSymbols>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <!-- Suppress tfm compatibility error when targeting net461 -->
+    <SuppressTfmSupportBuildErrors>true</SuppressTfmSupportBuildErrors>
```

Comment:
I thought this property already exist in common.props. Why isn't it already working?

Reply:
Oh never mind, other tests target net462 rather than net461. This is okay then.